### ISSUE TITLE
Invalid start state for planners

### ISSUE THREAD

**Issue by brianhou** [personalrobotics/herb_description#41]
The line https://github.com/personalrobotics/herb_description/blob/master/robots/herb_base.urdf.xacro#L370 seems to make the start state in the pantry-loading demo invalid. However, reverting that line to the way it is in https://github.com/personalrobotics/herb_description/commit/adfc50818b6070461fda3140eafe485a94032e8b at least makes the start state for planning valid.

**Comment by Shushman** [comment]
I have tried to address this (temporarily) in #42

**Comment by mkoval** [comment]
I don't understand why this causes planners to fail. What ends up being in collision?

**Comment by Shushman** [comment]
@pengjujin Could you shed some more light on this and maybe address Mike's question? I don't quite recall exactly what was happening. I just did what you suggested to fix it.

**Comment by pengjujin** [comment]
If I remembered this correctly, the original model had an incorrect frame for the Kinect sensor, so the perceived points would be in the wrong position relative to the robot. The first fix I tried was to change the rigid transformation between the Kinect link and the head link, but that rotated the Kinect model into the body which caused a self-collision during planning. The second fix I asked Shushman to do is essentially adding a fake link transformation in the urdf so the final transformation is correct without moving the Kinect model. The most correct fix should be changing the actual model of HERB.
